_packitpatch

```diff
@@ -39,7 +39,7 @@ patch_name=$(basename $patch_path)
 patch_args=$(echo ${@:3} | sed -e 's/ -b//' -e 's/--backup//')
 
 base_commit=$(git rev-parse HEAD)
-temp_branch="apply-${patch_name}"
+temp_branch="apply-$(echo ${patch_name} | sed -e 's/[^0-9a-z]/-/ig' -e 's/-\+/-/g' -e 's/^-//' -e 's/-$//')"
```

Comment:
Can we have a comment here about what the `sed` is good for? Thanks

Reply:
Done.